HOWTO.md

```diff
@@ -580,8 +589,11 @@ Please mind how the configuration file must be specifically named (e.g. `-` VS
    Files/serviceinfo-api-server.yml](#serviceinfo-api-serveryml), and place it either in
    `/usr/share/fdo`, `/etc/fdo/` or `/etc/fdo/serviceinfo-api-server.conf.d/`. The
    paths will be checked in that same order.
+      
+   Additionally, set up the `SERVICEINFO_API_SERVER_CONF` environment variable to point
+   to the path of the configuration file if running in a dev container.
 
-2. Execute `fdo-serviceinfo-api-server` or run it as a service, see sample file
+3. Execute `fdo-serviceinfo-api-server` or run it as a service, see sample file
```

Comment:
2

@rdotjain , can you fix this in another PR? thanks

Reply:
thanks for the review! but this seems like an issue with gh diff, because if we view the actual file the bullet points remain intact

Reply:
It doesn't look like an issue with gh diff. When the file is rendered in html, the markdown to html transpiler is smart enough to correctly number the lists, but if you see the file in raw format (markdown) you'll see that the numbers are not correct.

Reply:
@rdotjain please take a look

Reply:
Thanks for pointing it out! Fixed in #576